pkg/util/grpc/encoding/snappy/snappy.go

```diff
@@ -0,0 +1,81 @@
+package snappy
```

Comment:
So many nested pkg names feel so "Java"-like. Any downside to having it directly in `pkg/util/grpc/encoding` package?

Reply:
Importing this package have a side effect: `encoding.RegisterCompressor(newCompressor())`
So I prefer not having  it directly in `pkg/util/grpc/encoding`